openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -0,0 +1,17 @@
+{% spaceless %}
+{% load i18n %}
+<ol class="submission__answer__display__content">
+    {% for part in answer.parts %}
+        <li class="submission__answer__part">
+            <article class="submission__answer__part__prompt">
+                {{ part.prompt.description|linebreaks }}
+            </article>
+            <div class="submission__answer__part__text">
+                <div class="submission__answer__part__text__value">
+                    {{ part.text|linebreaks }}
```

Comment:
Add in a context header here (like line 58 in oa_response.html), but the header should read "Peer Response:". I think it would help to make this header visible on the page as well to lead reviewers to the part(s) they should be assessing.